src/tests/Navbar.test.js

```diff
@@ -31,6 +31,19 @@ describe('Navbar Component', () => {
     expect(wrapper.find('LogoutAction')).toBeTruthy();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should toggle the dropdown', () => {
+    const props = {
+      loggedIn: true,
+      history: { push: jest.fn() },
+      logoutAction: jest.fn(),
+    };
+    wrapper = shallow(<Navbar.WrappedComponent store={store} {...props} />);
+    const instance = wrapper.instance();
+    expect(wrapper.state().isOpen).toBe(false);
+    instance.toggle();
+    expect(wrapper.state().isOpen).toBe(true);
```

Comment:
@AGMETEOR can you look into this testing block to test properly? like simulating a click on the drop-down.

Reply:
@AGMETEOR, Adding another test to simulate the click on the dropdown would increase the confidence that our dropdown is working as expected.

Reply:
Let me do that.

Reply:
@muhozi I have written a test that takes care of the click.